docs/developer/upgrade-guide/3-1-to-3-2.mdx

```diff
@@ -4,7 +4,11 @@ title: Saleor 3.2
 
 ## Preparation
 
-Version 3.2 introduces some breaking changes necessary to ensure the highest quality of our API. There were no changes in public APIs, so storefronts do not need any changes to be compatible.
+Version 3.2 introduces some breaking changes necessary to ensure the highest quality of our API.
+
+## JWT token expiration
+
+Since Saleor 3.2, JWT token expiration is enabled by default. The default expiration time is 5 minutes and can be controlled server-side with `JWT_TTL_ACCESS` environment variable. See [Refreshing an access token](developer/users.mdx#refreshing-an-access-token) to learn how to refresh the token in your client app.
```

Comment:
```suggestion
Since Saleor 3.2, JWT token expiration time is set to 5 minutes and can be controlled server-side with `JWT_TTL_ACCESS` environment variable. See [Refreshing an access token](developer/users.mdx#refreshing-an-access-token) to learn how to refresh the token in your client app.
```